### DISCUSSION TITLE
ColorRect have wrong icon

### DISCUSSION THREAD

**Discussion by MeBadDev** [godotengine/godot-proposals#5456] [GUI]
![image](https://user-images.githubusercontent.com/94725241/190949303-87e8fa2f-9568-4eef-82c5-67a2dbf1e452.png)

AFAIK ColorRect doesn't support gradient without shader

**Comment by dalexeev** [comment]
![](https://user-images.githubusercontent.com/47700418/191011230-29f4a3c3-6ce4-4c63-8a62-2330160996e0.png)

It's not a skeleton, it's a skull. :smile:

Icons are not required to faithfully depict the appearance of a node. Although, there is some sense in your remark. But I'm not sure there is any other way to abstract the concept of color on such a small icon other than using different colors.

The `Color` type has this icon:

![](https://user-images.githubusercontent.com/47700418/191012101-cc87129b-86f0-4865-b890-de558ab17124.png)

But I'm not sure that if we make these letters even smaller and fit them into a rectangle, the result will be better.
